.changeset/brown-houses-love.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Having multiple changeset files with same description will add multiple changelog entries. Let's keep a single file as the description is same.

Reply:
Fixed with 1d3ef8fd4f615de27060c959ea91a56ca1f3158f